### PR TITLE
ND: add votes scraping to bills scraper

### PR DESCRIPTION
**Pull request by braykuka**
[1242](https://github.com/openstates/issues/issues/1242)

### CODE REVIEW

**Comment by braykuka** [comment]
@jessemortenson Please review it.

**Comment by jessemortenson** [comment]
Thanks, Bray, I think this is looking good. I have one request: can you please add logic to get the voter's full name from the voter's profile webpage? So instead of `Larsen` the voter name provided would be `Doug Larsen` (from https://ndlegis.gov/biography/doug-larsen). 

Please try to avoid including a title like "Representative" or "Senator" in the name though.

I think you'll want to implement a simple cache for those requests (eg `{ url: full_name }`) so we are not making 50-100 additional requests per vote event.

**Comment by braykuka** [comment]
@jessemortenson I've added the logic to get the full name. Please let me know if you have issues. Thanks.

File: scrapers/nd/bills.py [hunk 1]
```diff
@@ -30,6 +34,32 @@ def create_source_url(self):
             f"{assembly_session_id}-{year}/data/bills.json"
         )
 
+    def get_voter_name_from_url(self, url: str) -> str:
+        """
+        Description:
+            Get the full name from URL
```
[the remark this answers is not in the document]
Comment (braykuka) [reply scrapers/nd/bills.py:42]: @jessemortenson I've added this new logic but also updated the old logic too. Please review it and let me know if you want to remove the old logic. Thanks.

**Pull request merged**